vue/components/ui/organisms/details/details.vue

```diff
@@ -68,6 +68,7 @@
             <div class="container-header">
                 <div class="header-buttons">
                     <slot name="header-buttons">
+                        <slot name="header-buttons-extra" />

```

Comment:
header buttons before
and header buttons after

Reply:
Done!